scripts/nfsd-photon-v2.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Change the documentation to reflect NFS PV support is available on Photon as well. It currently says not supported.